canonical-websites/www.ubuntu.com
# Updated and moved contact-us pages

## Done

- Updated and moved contact-us pages

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [contact us](http://0.0.0.0:8001/contact-us), [form](http://0.0.0.0:8001/contact-us/form) and [thank-you]http://0.0.0.0:8001/contact-us/form/thank-you)
- compare to the [copy docs](https://drive.google.com/drive/folders/0Bw_yUXzV8CSHYjJ6VkNuanVWQVE?usp=sharing)

## Issue / Card

Fixes #2746
